src/client/src/assets/i-close.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="12" height="12" viewBox="0 0 12 12" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Should we give an alt tag to this svg?

Reply:
yup - good catch! I forgot to add it in since I didn't use these but will need it for another pr